Iteración 1/README.md

```diff
@@ -0,0 +1 @@
+*** Estragegias De Branching.docx
```

Comment:
El nombre no seria Estrategias?
Por otro lado esto sería lo único que contiene el readme?
Entendería que el readme en nuestro caso deberá sustituirla documentación que se entrega normalmente, o sea que debería haber título y demás (hasta tabla de contenidos posiblemente).
Adicionalmente falta alguna referencia a la definition of done en el readme

Reply:
Por otro lado quizás estaría bueno poner los documentos en sub-carpetas (si aplica) categorizados como corresponda